_data/changelogs/about-download.yml

```diff
@@ -0,0 +1,9 @@
+title: Download
+type: documentation
+changelogURL:
+items:
+  - date: NNNN-NN-NN
+    summary: Added note about incorrect package dates in downloadable USWDS package.
+    affectsGuidance: true
```

Comment:
```suggestion
```

Lets only include this one when there is a meaningful guidance change (ie something that would change how we suggest teams do something, like use a component). I think this copy clarification doesn't pass that test.